composer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "joomla/coding-standard",
+    "description": "Joomla! Coding Standard",
+    "keywords": ["joomla", "coding standard", "phpcs"],
+    "homepage": "https://github.com/joomla/coding-standards",
+    "license": "GPL-2.0+",
+    "require": {
+        "php": ">=5.3.10",
+        "squizlabs/php_codesniffer": "1.*"
```

Comment:
Are you sure, that standard doesn't work with PHP_CodeSniffer 2.x? If so the version constrain should be `~1.5` instead.